Mage.Tests/src/test/java/org/mage/test/cards/abilities/keywords/EvokeTest.java

```diff
@@ -41,8 +41,16 @@ public void testCreatureComesIntoPlay() {
 
         castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Shriekmaw");
         setChoice(playerA, true);
-        // addTarget(playerA, "Silvercoat Lion"); Autochosen, only target
+        setChoice(playerA, ""); //Stack triggers
```

Comment:
Why it empty?

Reply:
There's the creature's ETB effect and the evoke ETB effect going on the stack at the same time. The order the two triggers are stacked doesn't matter, so empty string is fine.

Reply:
Use real trigger name instead. Don’t use empty choice commands, it must be restricted for usage.

Reply:
`setChoice(playerA, "");` has been used in multiple other tests to indicate that triggers need to be stacked but order doesn't matter. Should I change all of those as well?

Reply:
Yes, it can be fixed out of that PR (if there are small amount then can be fixed here).

Reply:
If the triggers being stacked have identical text is the empty string acceptable?

Reply:
Yes, it’s ok. Just make sure double effects/triggers raised by design, not by error (e.g. duplicated effects).

Reply:
Well, it works for triggers, for replacement effects I can't actually order them because the random ID is before the effect text. I'll fix the triggers and leave replacement effect ordering for someone else.

Reply:
After going through the tests, it seems I was mistaken, All other examples of that pattern being used with trigger stacking actually were for ordering triggers with the same text, so I've only changed my own test here.